services/web/server/tests/unit/with_dbs/03/wallets/test_wallets_groups.py

```diff
@@ -0,0 +1,120 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+# pylint: disable=unused-variable
+# pylint: disable=too-many-arguments
+
+
+from collections.abc import AsyncIterator, Iterator
+
+import pytest
+import sqlalchemy as sa
+from aiohttp import web
+from aiohttp.test_utils import TestClient
+from pytest_simcore.helpers.utils_assert import assert_status
+from pytest_simcore.helpers.utils_login import NewUser, UserInfoDict
+from simcore_postgres_database.models.wallets import wallets
+from simcore_service_webserver.db.models import UserRole
+from simcore_service_webserver.projects.models import ProjectDict
+
+
+@pytest.fixture()
+def wallets_clean_db(postgres_db: sa.engine.Engine) -> Iterator[None]:
+    with postgres_db.connect() as con:
+        yield
+        con.execute(wallets.delete())
+
+
+@pytest.mark.parametrize("user_role,expected", [(UserRole.USER, web.HTTPOk)])
+async def test_wallets_groups_full_workflow(
+    client: TestClient,
+    logged_user: UserInfoDict,
+    user_project: ProjectDict,
+    expected: type[web.HTTPException],
+    wallets_clean_db: AsyncIterator[None],
+):
+    # create a new wallet
+    url = client.app.router["create_wallet"].url_for()
+    resp = await client.post(
+        f"{url}", json={"name": "My first wallet", "description": "Custom description"}
+    )
+    added_wallet, _ = await assert_status(resp, web.HTTPCreated)
+
+    # check the default wallet permissions
+    url = client.app.router["list_wallet_groups"].url_for(
+        wallet_id=f"{added_wallet['wallet_id']}"
+    )
+    resp = await client.get(f"{url}")
+    data, _ = await assert_status(resp, web.HTTPOk)
+    assert len(data) == 1
```

Comment:
TIP: since you have a model for the response, just use it here as well . This way you test schema models as well